packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -106,11 +106,14 @@ export class DocumentDeltaConnection
     /**
      * @param socket - websocket to be used
      * @param documentId - ID of the document
+     * @param logger - for reporting telemetry events
+     * @param driverVersion - driverVersion to use when raising errors
      */
     protected constructor(
         protected readonly socket: SocketIOClient.Socket,
         public documentId: string,
         logger: ITelemetryLogger,
+        private driverVersion: string,
```

Comment:
I'm sneaking `driverVersion` onto the `logger` as well in `createOdspLogger`, I could remove this param and just pull it off the logger if that's better, like I do for parallel requests code (where it was too awkward to plumb "honestly")

Reply:
Nope no duplicate logging.  Also I switched this and just using the snuck in version on the logger.